e2e/runner/cypress-runner-backend.js

```diff
@@ -27,7 +27,8 @@ const CypressBackend = {
         "-Dh2.bindAddress=localhost", // fix H2 randomly not working (?)
         "-Djava.awt.headless=true", // when running on macOS prevent little Java icon from popping up in Dock
         "-Duser.timezone=US/Pacific",
-        `-Dlog4j.configurationFile=file:${__dirname}/../../frontend/test/__runner__/log4j2.xml`,
+        // if you comment this line 👇 you can get (very noisy) backend console logs in the terminal for e2e tests
+        // `-Dlog4j.configurationFile=file:${__dirname}/../../frontend/test/__runner__/log4j2.xml`,
```

Comment:
Did you intentionally leave this commented in order to have BE logs in CI?

Reply:
yeah, kept this in for visibility, don't want to merge it this way 🔥